apps/web/pages/team/[slug]/[type].tsx

```diff
@@ -57,16 +57,38 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   const { rescheduleUid } = context.query;
   const { ssrInit } = await import("@server/lib/ssr");
   const ssr = await ssrInit(context);
-  const { currentOrgDomain, isValidOrgDomain } = orgDomainConfig(context.req.headers.host ?? "");
+  const { currentOrgDomain, isValidOrgDomain } = orgDomainConfig(
+    context.req.headers.host ?? "",
+    context.query.orgSlug
+  );
 
   const team = await prisma.team.findFirst({
     where: {
-      slug: teamSlug,
-      parent: isValidOrgDomain
-        ? {
-            slug: currentOrgDomain,
-          }
-        : null,
+      OR: [
+        { slug: teamSlug },
+        {
+          metadata: {
+            path: ["requestedSlug"],
```

Comment:
Needed to make sure we also selected an unpublished organization to not end up in a 404 page